packages/common/src/HardhatConfig.ts

```diff
@@ -91,6 +91,7 @@ export function getHardhatConfig(
         blockGasLimit: 15_000_000,
         timeout: 1800000,
         testBlacklist,
+        allowUnlimitedContractSize: true,
```

Comment:
Is this necessary? Does this mean that we don't know if our contracts are over the limit when running against a test network?

Reply:
its just for the `VotingV2Test` that is now above the byte limit.